docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md

```diff
@@ -38,323 +47,79 @@ You'll need a 1Password API token and your customer-specific 1Password domain (f
-2. **Started**. A collection task is created on the Hosted Collector.
-3. **Initialized**. The task configuration is complete in Sumo Logic.
-4. **Authenticated**. The Source successfully authenticated with 1Password.
-5. **Collecting**. The Source is actively collecting data from 1Password.
-
-If the Source has any issues during any one of these states, it is placed in an **Error** state.
-
-When you delete the Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
-
-On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events) to investigate issues with collection. You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.<br/> ![1password state.png](/img/send-data/1password-state.png)
-
-Hover your mouse over the status icon to view a tooltip with details on the detected issue.<br/> ![1password](/img/send-data/health_error_generic.png)
-
-## Create a 1Password Source
-
-When you create a 1Password Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Create a Hosted Collector](/docs/send-data/hosted-collectors#Create_a_Hosted_Collector).
-
-To configure a 1Password Source:
-
+### Source configuration
 1. In Sumo Logic, select** Manage Data > Collection > Collection**.
-2. On the Collectors page, click **Add Source** next to a Hosted** **Collector.
-3. Select **1Password**.<br/><img src={useBaseUrl('img/send-data/1password-source-icon.png')} alt="1password-source-icon.png" width="150" />
-4. Enter a **Name** for the Source. The **description** is optional.<br/> ![1password-input](/img/send-data/1password-input.png)
-5. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category [metadata](/docs/search/get-started-with-search/search-basics/built-in-metadata) is stored in a searchable field called `_sourceCategory`.
-6. **Forward to SIEM**. Check the checkbox to forward your data to [Cloud SIEM Enterprise](/docs/cse). When configured with the **Forward to SIEM** option, the following metadata fields are set:
-
-| Field Name | API | Value |
-|:---|:---|:---|
-| `_siemVendor` | Sign-in, Item | `1Password` |
-| `_siemProduct` | Sign-in, Item | `1Password` |
-| `_siemFormat` | Sign-in, Item | `JSON` |
-| `_siemEventID` | Sign-in | `signin-{{category}}` |
-| `_siemEventID` | Item | `item_usage-{{action}}` |
-
-7. (Optional) **Fields**. Click the **+Add** link to add custom log metadata [Fields](/docs/manage/fields).
+1. On the Collectors page, click **Add Source** next to a Hosted** **Collector.
+1. Select **1Password**.<br/><img src="https://app_icons.s3.amazonaws.com/1Password.png" alt="1password-source-icon.png" width="150" />
```

Comment:
Do you need the image file here? It looks strange here in the middle of the procedure.

Reply:
Happy to remove it. I left it as it seemed other C2Cs had it. I'm all for less screenshots. What would you suggest @jpipkin1?

Reply:
@yleiferman, remove the screenshot. It's not needed.

Reply:
@jpipkin1 screenshot removed.